cli/src/main/scala/aqua/run/RunOpts.scala

```diff
@@ -25,15 +28,42 @@ object RunOpts {
         "/dns4/kras-00.fluence.dev/tcp/19001/wss/p2p/12D3KooWR4cv1a8tv7pps4HH6wePNaK6gf1Hww5wcCMzeWxyNw51"
       )
 
-  val funcNameOpt: Opts[String] =
+  def spanToId: Span.S ~> Id = new (Span.S ~> Id) {
+
+    override def apply[A](span: Span.S[A]): Id[A] = {
+      span._2
+    }
+  }
+
+  val funcOpt: Opts[(String, List[LiteralModel])] =
     Opts
       .option[String]("func", "Function to call with args", "f")
+      .mapValidated { str =>
+        CallArrowExpr.funcOnly.parseAll(str) match {
+          case Right(f) =>
+            val expr = f.mapK(spanToId)
+            val hasVars = expr.args.exists {
+              case VarLambda(_, _) => true
+              case _ => false
+            }
+            if (hasVars) {
+              Validated.invalidNel("Function must have only literals as variables")
```

Comment:
Could it contain constants from the compiled file?

Reply:
`Function can have only literal arguments, no variables or constants allowed at the moment`